e2e/bin/update

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if [ "$1" == "" ]; then
+    echo "./upload.sh <source> <destination>"
+    exit 1
+fi
+
+if [ "$2" == "" ]; then
```

Comment:
I think you can do something like `if [[ "$1" == "" || "$2" == "" ]]; then` or `if [[ $# -ne 2 ]]` as well.